modules/mono/editor/Godot.NET.Sdk/Godot.NET.Sdk/Godot.NET.Sdk_PackageVersion.txt

```diff
@@ -0,0 +1 @@
+4.0.0-dev3
```

Comment:
CI checks show a missing newline here (though GitHub doesn't).

Reply:
Luckily I made the code trim that file content, so adding a new line at the end shouldn't break anything.

Reply:
Yup, the line is trimmed correctly in all 3 places it's used.